lib/zfs.js

```diff
@@ -307,6 +307,30 @@ function clone(opts, cb) {
 
 /*
 *
+*  Mount the specified dataset to the mountpoint (have been set before mount)
+*
+*  PARAMS:
+*  opts: {
+*    dataset: string // the name of the zfs dataset (must exist)
+*    overlay: boolean // whether use overlay mode
+*  }
+ */
+function mount(opts, cb) {
+    "use strict";
+    
+    var params = [ 'mount' ];
+    
+    if (opts.overlay) {
+        params.push('-O');
+    }
+    
```

Comment:
`-O` is fairly exotic. If we support that, should we not support generic `-o` options?

`{ options: {mountpoint: "/foo"}}` 

for example?